extensions/sql-migration/src/service/features.ts

```diff
@@ -104,7 +104,8 @@ export class SqlMigrationService extends MigrationExtensionService implements co
 			startTime,
 			endTime,
 			includePreviewSkus,
-			databaseAllowList
+			databaseAllowList,
+			isPremiumSSDV2Enabled: true,
```

Comment:
I like that you added the propertyname here, I suggest improving the rest of it, by adding the parameters to the other properties being set in the objec/interface being created/filled out for improved readability.

Reply:
For others variable name was quite explicite and the last one i added was literal so made it named parameter. Taken your comment here but now it is looking repetative but it will help countering issue due to order.